tests/compact-manifest.jsonld

```diff
@@ -957,6 +957,15 @@
       "input": "compact/0113-in.jsonld",
       "context": "compact/0113-context.jsonld",
       "expect": "compact/0113-out.jsonld"
+    }, {
+      "@id": "#t0114",
+      "@type": ["jld:PositiveEvaluationTest", "jld:CompactTest"],
+      "name": "Reverse term with property based indexed container",
+      "purpose": "With reverse term using @container: @index and @index set to a property, ensure round-tripping from expaned form",
+      "input": "expand/0131-out.jsonld",
```

Comment:
I'd suggest changing this to be consistent with the independent files per-test and naming pattern used for all other tests.  Copy the files from `expand` to `compact` and rename them to match the param in/context/out pattern.  I see it's doing a round-tripping thing, but the change from the normal pattern could be confusing to others and our future selves.

Reply:
> (Note: the round-tripping compaction test reuses the expand test data. That does not seem to be the practise; is there any rationale against that?)

Missed that comment... one reason is to be consistent.  Reusing cross algorithms, mixing up in/out/context naming... it will work but it's not straightforward.  I think it's better to just be clear what the files are and for which test and which params.

Reply:
Agreed! Addressed in a6a1dcb8af02b7a589e85be5a6c391c71f18412d.